modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/impl/RestAPIStoreImpl.java

```diff
@@ -1153,7 +1147,7 @@ public SubscriptionDTO subscribeToAPI(String apiID, String appID, String tier) t
      * @return
      * @throws ApiException
      */
-    public APIListDTO getAPIListFromStoreAsAnonymousUser(String tenantDomain) throws ApiException {
+    public APIListDTO getAPIListFromStoreAsAnonymousUser(String tenantDomain, String storeURL) throws ApiException {
```

Comment:
I don't think this change is needed as the store URL can be passed through the constructor.

Reply:
Fixed in 09e067607b550f97373afb9510ea71424bed8a67